### DISCUSSION TITLE
Using the docker-compose for Postgres with MSF

### DISCUSSION THREAD

**Discussion by lonniev** [rapid7/metasploit-framework#15322] [Q&A]
There's a lot of info available for how to set up Postgres for Metasploit when either the Postgres service is on the same host as the Metasploit framework or when there is a Linux shell with root access to all the Metasploit tools (like the key msfdb tool).

But where is the guide for using the Docker-Compose combination (see the file in this git repo) of the images for Metasploit-framework and for Postgres such that the admin can hop into the Metasploit container and shell out of the Msfconsole to get to the msfdb commands to do the initial msf tables setup for the Postgres in the other container?

How are we supposed to gracefully run/start/enter the "ms" container to complete that bit of msf/postgres setup?

**Comment by lonniev** [comment]
aha, one way to get to a shell from within Msfconsole is to run `bash`.

**Reply by adfoster-r7** [reply]
Let me know if this script works for your workflow :+1:

https://github.com/rapid7/metasploit-framework/issues/15144#issuecomment-854001016
